apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -194,4 +194,8 @@
     public static final OfficialComponent APACHE_CXF = new OfficialComponent(105, "Apache-CXF");
 
     public static final OfficialComponent DOLPHIN_SCHEDULER = new OfficialComponent(106, "dolphinscheduler");
+
+    public static final OfficialComponent JSON_RPC_CLIENT = new OfficialComponent(107, "JsonRpcClient");
+
+    public static final OfficialComponent JSON_RPC_SERVER = new OfficialComponent(108, "JsonRpcServer");
```

Comment:
We usually share one component ID at both client and server sides.

Reply:
Do you have specific reason of needing both?

Reply:
> Do you have specific reason of needing both?

Got it, I will change it.

Reply:
> Do you have specific reason of needing both?

fixed, add it in one Component